src/spotted/handlers/job_handlers.py

```diff
@@ -62,3 +62,27 @@ async def db_backup_job(context: CallbackContext):
             )
         except Exception as ex:  # pylint: disable=broad-except
             await context.bot.send_message(chat_id=admin_group_id, text=f"✖️ Impossibile effettuare il backup\n\n{ex}")
+
+
+async def clean_warned_users():
+    """Job called each day at 05:00 utc.
+    Removed users who have been warned for longer than setting duration
+    Args:
+        context: context passed by the jobqueue
+    """
+    warn_expiration = datetime.now() + timedelta(days=Config.post_get("warn_after_days"))
+    DbManager.delete_from(
+        table_name="warned_users",
+        where="warn_time > %s",
+        where_args=(warn_expiration,),
+    )
```

Comment:
It depends on how you implement `warn_time`, but this should probably be a `-` and `warn_time < %s`.

Reply:
not sure 🤔